### PR TITLE
Indi bresserexos2 v900 hotfix

### PR DESCRIPTION
**Pull request by kneo**
Errors were made and fixed in time and location handling. 
This PR fixes: 
- 360° complement longitude handling for negative (westward) values.
- libnova now handles time and zone handling and should no longer provide wrong times to the handbox.

### CODE REVIEW

File: indi-bresserexos2/BresserExosIIGoToDriver.cpp [hunk 1]
```diff
@@ -303,30 +303,46 @@ bool BresserExosIIDriver::SetTrackingEnabled(bool enabled)
...
+    //orientation of the handbox is: 
+    //negative longitude is west of greenich
+    //positive longitude is east of greenich
+    //kstars sends 360 complements for negatives
+    //this sole case needs to be corrected:
+    double realLongitude = longitude;
+    
+    if(realLongitude>180)
+    {
+        realLongitude = -(360-realLongitude);
```
Comment (knro) [inline indi-bresserexos2/BresserExosIIGoToDriver.cpp:340]: Use instead reallongitude -= 360

File: indi-bresserexos2/BresserExosIIGoToDriver.cpp [hunk 2]
```diff
@@ -303,30 +303,46 @@ bool BresserExosIIDriver::SetTrackingEnabled(bool enabled)
 bool BresserExosIIDriver::updateTime(ln_date *utc, double utc_offset)
 {
     INDI::Telescope::updateTime(utc, utc_offset);
```
Comment (knro) [inline indi-bresserexos2/BresserExosIIGoToDriver.cpp:305]: No need to call this.

**Review by knro — CHANGES_REQUESTED** [review]
Minor changes.

File: indi-bresserexos2/BresserExosIIGoToDriver.cpp (hunk 1 above)
Comment (kneo) [reply indi-bresserexos2/BresserExosIIGoToDriver.cpp]: applied as requested in latest commit.

File: indi-bresserexos2/BresserExosIIGoToDriver.cpp (hunk 2 above)
Comment (kneo) [reply indi-bresserexos2/BresserExosIIGoToDriver.cpp]: applied as requested in latest commit.

**Review by knro — APPROVED** [review]

**Pull request merged**